### DISCUSSION TITLE
Object scoped actions on yank/select/etc

### DISCUSSION THREAD

**Discussion by TheFedaikin** [LazyVim/LazyVim#1880] [Q&A] [answered]
Hey! Thanks for creating this awesome configuration.
I wanted to replicate some of LazyVim functionality with slimmer config version of mine but couldn't crack this one - I couldn't get where the object scoped actions (`Object scope` and `Object scope with border`)
I tried to look where the `i Object scope with border` in the screenshot comes from (I assumed it was from `mini.ai`) but couldn't find anything in the `LazyVim` config [here](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/coding.lua#L188). Now I think it's one of the loops or some downstream plugin, but couldn't find it at all. Where is this coming from?
<img width="611" alt="Screenshot 2023-10-26 at 5 12 58 PM" src="https://github.com/LazyVim/LazyVim/assets/24460783/6e42eb13-1207-4461-843c-d9dcd9d8b5f2">

**Comment by dpetka2001** [comment, marked as answer]
[mini.indentscope](https://github.com/echasnovski/mini.indentscope) I believe.

**Reply by TheFedaikin** [reply]
Thank you, that's exactly it.
